code/modules/mob/living/silicon/robot/robot_defines.dm

```diff
@@ -202,10 +202,26 @@
 	cell = /obj/item/stock_parts/cell/hyper
 	radio = /obj/item/radio/borg/syndicate
 
+/mob/living/silicon/robot/model/syndicate/Initialize(mapload)
+	laws = new /datum/ai_laws/syndicate_override()
+	laws.associate(src)
+	. = ..()
+	addtimer(CALLBACK(src, PROC_REF(show_playstyle)), 5)
```

Comment:
A) needs `SECONDS` define or `DECISECONDS` or something idk
B) could we invoke this before we call parent so we can have a clean `return ..()` at the end here?

Reply:
It's probably preferred that it comes after parent call

Reply:
ehhhhhhhhhhhhhhhh i guess but i wish it didn't have to look ugly like that. oh well